taxonomies/categories.txt

```diff
@@ -35244,6 +36290,13 @@ nl:Groentencakes
 en:Stuffed vegetables
 fr:Légumes farcis
 
+<en:Stuffed vegetables
+en:Stuffed vegetables (excluding tomatoes)
```

Comment:
parenthesis are not supported in the taxonomy, they will be treated as a synonym : "excluding tomatoes" will be a synonym of "stuffed vegetables", which is not what we want.

For this case:

Just put "stuffed vegetables" with the corresponding CIQUAL category, and then under "stuffed tomatoes" you can put the more specific ciqual category.

Reply:
Compris, merci pour l'explication !
Corrigé dans la prochaine PR